packages/contracts-bedrock/contracts/L1/SystemConfig.sol

```diff
@@ -79,52 +115,66 @@ contract SystemConfig is OwnableUpgradeable, Semver {
     event ConfigUpdate(uint256 indexed version, UpdateType indexed updateType, bytes data);
 
     /**
-     * @custom:semver 1.0.1
+     * @custom:semver 1.1.0
      *
      * @param _owner             Initial owner of the contract.
      * @param _overhead          Initial overhead value.
      * @param _scalar            Initial scalar value.
      * @param _batcherHash       Initial batcher hash.
      * @param _gasLimit          Initial gas limit.
      * @param _unsafeBlockSigner Initial unsafe block signer address.
+     * @param _config            Initial resource config.
      */
     constructor(
         address _owner,
         uint256 _overhead,
         uint256 _scalar,
         bytes32 _batcherHash,
         uint64 _gasLimit,
-        address _unsafeBlockSigner
-    ) Semver(1, 0, 1) {
-        initialize(_owner, _overhead, _scalar, _batcherHash, _gasLimit, _unsafeBlockSigner);
+        address _unsafeBlockSigner,
+        ResourceConfig memory _config
+    ) Semver(1, 1, 0) {
+        initialize({
```

Comment:
Fine, I'll let it slide for this one :wink:

Reply:
:P <3